### ISSUE TITLE
[Feature Request] Ability to cycle through available video sources / webcams

### ISSUE THREAD

**Issue by MagicFab** [telegramdesktop/tdesktop#10015]
I am using a Jabra Panacast (panoramic, auto-pan/zoom whit tracking features), and HD webcam and a laptop webcam.

I'd like to have a button available on Telegram desktop to cycle through the available webcams while on a call.

Right now I have to got to the Calls settings menu. I am using Debian 10, in the Cheese webcam application this is implemented through a button that cycles through webcams.

**Comment by s-light** [comment]
its a feature Request...
not yet implemented.

but for me at least the switching of the camera during a call worked last time i tested this :-)
also this is cumbersome to achive as the video-device-setting is hidden very very deep in the settings....

**Comment by djmattyg007** [comment]
Just tried a video call through the desktop app for the first time and could not get it to use the correct webcam device :( Please let us choose manually.

**Comment by s-light** [comment]
@djmattyg007 you can manually select the cam:
Go to
- Settings 
- Advanced
![image](https://user-images.githubusercontent.com/1340319/125238275-4cdff300-e2e7-11eb-80cd-f92a77921a73.png)
System Integration:
- Call Settings
- Input Device `*click*`
![image](https://user-images.githubusercontent.com/1340319/125238420-87499000-e2e7-11eb-8663-09f53e065842.png)

**Comment by djmattyg007** [comment]
It did not occur to me that "Call Settings" was a button I could click on, because it's in the same subgroup as a list of checkboxes. It looked like a heading for those checkboxes.

Thankyou.

**Comment by MagicFab** [comment]
Why is this closed ? The original request acknowledges the "Call Settings" way of switching cameras which is not easy while being on a call. @Aokromes can you please leave this open for tracking ?

**Comment by john-preston** [comment]
Yeah, I agree that it should be done somewhere inside the call / voice chat window.

**Comment by mirh** [comment]
Still to be added.

**Comment by djmattyg007** [comment]
Definitely not stale.

**Comment by s-light** [comment]
still valid.
